### PR TITLE
🐛 fix dynamic-sidecar inactivity response when user service reply is not formatted correctly

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

Added regression test to catch incorrect json formatted response form user services.
Sidecar will now mark the service as inactive if it cannot properly parse the reply from the user service.

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by mguidon — APPROVED** [review]
lgtm

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py [hunk 1]
```diff
@@ -112,6 +112,7 @@ async def get_containers_inactivity(
         ContainerExecContainerNotFoundError,
         ContainerExecCommandFailedError,
         ContainerExecTimeoutError,
+        json.JSONDecodeError,
```
Comment (bisgaard-itis) [inline services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py:115]: Would it make sense to write a different message in the logs depending on these cases, so that it is easier to go in and debug when it happens?

**Review by bisgaard-itis — APPROVED** [review]
Looks good to me. Just a single suggestion

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py (hunk 1 above)
Comment (GitHK) [reply services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py:115]: I think it's fine. It's used to give an idea about what happened. At most I can reword it to make it sound more generic.

Comment (GitHK) [reply services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py:115]: I think you have a good point now that I think about it. Let me make it a bit better.

Comment (GitHK) [reply services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py:115]: I think something like this would look much better
```
Could not parse command result 'This is an unparsable json response {}' as 'InactivityResponse'
Traceback (most recent call last):
  File "/home/silenthk/work/pr-osparc-fix-inactivity-error/services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py", line 126, in get_containers_inactivity
    return parse_raw_as(InactivityResponse, inactivity_response)
  File "pydantic/tools.py", line 74, in pydantic.tools.parse_raw_as
  File "pydantic/parse.py", line 37, in pydantic.parse.load_str_bytes
  File "/home/silenthk/.pyenv/versions/3.10.4/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/home/silenthk/.pyenv/versions/3.10.4/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/silenthk/.pyenv/versions/3.10.4/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```
